### PR TITLE
[SRVCOM-1666][MAIN] Fix version in csv urls and other broken ones

### PR DESCRIPTION
**Pull request by skonto**
Fix url version to point to latest (4.9). Still investigating if urls can be generated dynamically for the OCP version we are running on. The main question is what version of Serverless docs they should be reading? Potentially if someone is running on 4.6 with 1.20 he might want to read all Serverless doc releases up to 4.9.

### CODE REVIEW

File: olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -153,9 +153,9 @@ spec:
     # Further Information
     For documentation on OpenShift Serverless, see:
     - [Installation
-    Guide](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.6/html/serverless/administration-guide#installing-openshift-serverless)
+    Guide](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.9/html/serverless/administration-guide#installing-openshift-serverless)
```
Comment (maschmid) [inline olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml:156]: https://access.redhat.com/documentation/en-us/openshift_container_platform/4.9/html/serverless/install

Comment (skonto) [reply olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml:156]: I thought it was just the version things have changed then, it is more than that. :(

File: olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -153,9 +153,9 @@ spec:
     # Further Information
     For documentation on OpenShift Serverless, see:
     - [Installation
-    Guide](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.9/html/serverless/administration-guide#installing-openshift-serverless)
+    Guide](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.9/html/serverless/install)
     - [Getting
-    started](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.9/html/serverless/serverless-getting-started)
+    started](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.9/html/serverless/develop)
```
Comment (maschmid) [inline olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml:158]: We should change the title as well to match the new link then, 

"Develop Serverless Applications" maybe?

Comment (skonto) [reply olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml:158]: Yeah wfm.

**Comment by maschmid** [comment]
There's one more instance of a broken link on the CSV:

https://github.com/openshift-knative/serverless-operator/blob/main/templates/csv.yaml#L86

https://access.redhat.com/documentation/en-us/openshift_container_platform/4.9/html-single/serverless/index#serverless-getting-started

(the page exists, but not the anchor , could be replaced by   https://access.redhat.com/documentation/en-us/openshift_container_platform/4.9/html-single/serverless/index#install-serverless-operator  instead?

**Comment by skonto** [comment]
@maschmid will use `https://access.redhat.com/documentation/en-us/openshift_container_platform/4.9/html/serverless/index#install-serverless-operator` instead. Not picking the html-single path because it does not render correctly and want to comply with the rest of the urls.

**Comment by skonto** [comment]
@maschmid I think this is ready to go in.

File: templates/csv.yaml [hunk 3]
```diff
@@ -83,7 +83,7 @@ spec:
 
     The components provided with the OpenShift Serverless operator require minimum cluster sizes on
     OpenShift Container Platform. For more information, see the documentation on [Getting started
-    with OpenShift Serverless](https://access.redhat.com/documentation/en-us/openshift_container_platform/__OCP_TARGET__/html-single/serverless/index#serverless-getting-started).
+    with OpenShift Serverless](https://access.redhat.com/documentation/en-us/openshift_container_platform/__OCP_TARGET__/html/serverless/index#install-serverless-operator).
```
Comment (maschmid) [inline templates/csv.yaml:86]: Did you mean

https://access.redhat.com/documentation/en-us/openshift_container_platform/__OCP_TARGET__/html/serverless/install ?

(there's no install-serverless-operator anchor on the index page, https://access.redhat.com/documentation/en-us/openshift_container_platform/4.9/html/serverless/index#install-serverless-operator  open just the root of the index, same as  https://access.redhat.com/documentation/en-us/openshift_container_platform/4.9/html/serverless/index page )

Comment (skonto) [reply templates/csv.yaml:86]: The link should be https://access.redhat.com/documentation/en-us/openshift_container_platform/4.9/html/serverless/install#install-serverless-operator

**Pull request merged**